src/matlab2tikz.m

```diff
@@ -218,6 +222,7 @@ function matlab2tikz(varargin)
 ipp = ipp.addParamValue(ipp, 'extraCodeAtEnd', {}, @isCellOrChar);
 ipp = ipp.addParamValue(ipp, 'extraAxisOptions', {}, @isCellOrChar);
 ipp = ipp.addParamValue(ipp, 'extraTikzpictureOptions', {}, @isCellOrChar);
+ipp = ipp.addParamValue(ipp, 'extraTikzLegendOptions', {}, @isCellOrChar);
```

Comment:
While you at it, can you change the above to extraTikzPictureOptions, to stay consistend. I assume previously tikzpicture was considered one word.